src/platform/common/scriptPathUtils.ts

```diff
@@ -0,0 +1,50 @@
+import { inject } from 'inversify';
+import * as path from '../vscode-path/path';
+import { IExtensionContext, IScriptPathUtils } from './types';
+
+export class ScriptPathUtils implements IScriptPathUtils {
+    constructor(@inject(IExtensionContext) private readonly context: IExtensionContext) {}
+
+    get dataFrameLoading() {
```

Comment:
This file seems to be specific to node.js
As we're using `this.context.extensionPath`. 
However that means this cannot be used on the web and we want to be able to use this on the web.

Hence my suggestion is:
* use `this.context.extensionUri` instead of `context.extensionPath`
This changes the types from strings to Uri, which is a good thing as we want to avoid using strings for paths where ever possible, & use Uris instead.

Reply:
My concern then is that Uris need to convert to strings at some point, right? specially in these cases where we’re concatenating paths.